### ISSUE TITLE
eHighway Sweden | subgrid 89 

### ISSUE THREAD

**Issue by ryan-oreilly** [openENTRANCE/openentrance#149]
Should this subgrid be eHighway cluser 89_SE? It is defined as 88_SE which is already defined.

https://github.com/openENTRANCE/openentrance/blob/ec0b49e4746f38de9ca862dbea1258c230e016ff/definitions/region/ehighway.yaml#L553

**Comment by danielhuppmann** [comment]
Yes, I guess that tis was a typo by @sandrinecharousset, thanks for catching that. Can you please start a PR to fix it?
